packages/core/src/external/carequality/ihe-gateway-v2/ihe-gateway-v2-async.ts

```diff
@@ -9,7 +9,14 @@ import { makeLambdaClient } from "../../aws/lambda";
 import { Config } from "../../../util/config";
 import { processAsyncError } from "../../../util/error/shared";
 import { IHEGatewayV2 } from "./ihe-gateway-v2";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
 
+dayjs.extend(duration);
+
+const SLEEP_IN_BETWEEN_DOCUMENT_RETRIEVAL_REQUESTS = dayjs
+  .duration({ seconds: 1 })
+  .asMilliseconds();
```

Comment:
Its better to create the const as duration and convert to the unit needed on the place where we use it